include/sound/sof/info.h

```diff
@@ -126,4 +127,11 @@ struct sof_ipc_probe_support {
 	uint32_t reserved[2];
 } __packed;
 
+/* extended data: user abi version(s) */
+struct sof_ipc_user_abi_version {
+	struct sof_ipc_ext_data_hdr ext_hdr;
+
+	uint32_t abi_dbg_version;
```

Comment:
wondering whether we aren't overcomplicating things and whether we shouldn't stick to just one ABI version

Reply:
@lyakh No idea of the usage model of this, but this discussion needs to happen on the FW side PR as the people involved are not reading this.